src/Util/Crypto.php

```diff
@@ -267,13 +267,13 @@ private static function encryptAES256CBC($data, $key, $iv)
 
 	/**
 	 * Decrypt a string with 'aes-256-cbc' cipher method.
-	 * 
+	 *
 	 * Ported from Hubzilla: https://framagit.org/hubzilla/core/blob/master/include/crypto.php
-	 * 
+	 *
 	 * @param string $data
-	 * @param string $key   The key used for decryption.
-	 * @param string $iv    A non-NULL Initialization Vector.
```

Comment:
Those white spaces are actually wanted, so all the parameter comments are aligned.